domehunter/gRPC-server/hx2dome.proto_server.py

```diff
@@ -571,7 +573,25 @@ def serve(kwargs_dict):
                         to the TheSkyX.")
     parser.set_defaults(server_testing=False)
 
-    kwargs_dict = vars(parser.parse_args())
+    parser.add_argument('-y', '--yamlconfig',
+                        dest='yamlconfig',
+                        help="YAML file containing cofiguration details for \
```

Comment:
```suggestion
                        help="YAML file containing configuration details for \
```

Reply:
This line break probably also shows up weird.

Reply:
maybe, I haven't checked. Just added it because my editor gets fussy about pep8 line lengths